From 0e65f1427d040f24e80261f12bd0f884298f3e8d Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Wed, 21 Feb 2018 21:48:49 +0100
Subject: [PATCH 3/4] wifi: don't check if WIFI_DRIVER_STATE_CTRL_PARAM is
 readable

Checking if it's writable is enough because we're only writing to it.

Change-Id: Ib09ca2ee20b1555435b1d92d20465662ee7cf463
---
 libwifi_hal/wifi_hal_common.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libwifi_hal/wifi_hal_common.cpp b/libwifi_hal/wifi_hal_common.cpp
index aac65e1d4..49bb9d1af 100644
--- a/libwifi_hal/wifi_hal_common.cpp
+++ b/libwifi_hal/wifi_hal_common.cpp
@@ -104,7 +104,7 @@ int wifi_change_driver_state(const char *state) {
   if (!state) return -1;
 
   do {
-    if (access(WIFI_DRIVER_STATE_CTRL_PARAM, R_OK|W_OK) == 0)
+    if (access(WIFI_DRIVER_STATE_CTRL_PARAM, W_OK) == 0)
       break;
     usleep(200000);
   } while (--count > 0);
-- 
2.17.1

